python/lsst/verify/ap/pipedriver.py

```diff
@@ -0,0 +1,193 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import absolute_import, division, print_function
+
+__all__ = ["ApPipeParser", "run_ap_pipe"]
+
+import argparse
+
+import lsst.log
+from lsst.verify.ap.dataset import Dataset
+
+
+def _ingest_raws(dataset, repo_root):
```

Comment:
I couldn't think of any particular program state that would be guaranteed by having a `PipeWrapper` object sitting around, like there is with `Dataset` and `AutoJob`.

I guess I could create a class as a convenience wrapper for the function arguments, but that seems a bit gratuitous. It would also make it more difficult for other coders to implement the stubs and/or keep straight which information is needed for which pipeline step.